
<launch>

   
    <!-- 运行地图服务器，并且加载设置的地图-->
    <node name="obstable_detection" pkg="obstable_detection" type="obstable_detection_node" >

    <param name="safety_distance" value="1.0"/>
    <param name="safety_tolerance" value="5"/>

    </node>


<!-- 运行rviz -->
    <!--<node pkg="rviz" type="rviz" name="rviz" args="-d $(find aibee_robot)/rviz/obstable.rviz"/>-->

</launch>

